src/Microsoft.AspNet.Diagnostics/ErrorPageMiddleware.cs

```diff
@@ -51,14 +55,16 @@ public ErrorPageMiddleware([NotNull] RequestDelegate next, [NotNull] ErrorPageOp
             }
             catch (Exception ex)
             {
+                _logger.LogError("An unhandled exception has occurred while executing the request", ex);
```

Comment:
I noticed that `ErrorHandler` middleware already has some logging...not sure if we are looking for more.
https://github.com/aspnet/Diagnostics/blob/dev/src/Microsoft.AspNet.Diagnostics/ErrorHandlerMiddleware.cs#L37